packages/arb-token-bridge-ui/src/util/bridgeUiConfig.ts

```diff
@@ -128,7 +128,8 @@ export function getBridgeUiConfigForChain(chainId: number): BridgeUiConfig {
         network: {
           name: customChain ? customChain.name : 'Unknown',
           logo: '/images/OrbitLogo.svg'
-        }
+        },
+        nativeTokenData: customChain?.nativeTokenData
```

Comment:
Can we have the object extend from `...customChain` and overwrite `network` etc. that we want to be customized here?